olive/passes/onnx/graph_surgeries.py

```diff
@@ -447,6 +452,50 @@ def __call__(self, model: ModelProto):
         return self._add_zero_point(model, zero_point_value, zero_point_onnx_dtype, zero_point_np_dtype)
 
 
+class QuickGeluToSigmoid(Surgeon):
+    """Lower QuickGelu operator to standard ONNX operators.
+
+    QuickGelu pattern:
+    [Input] --> QuickGelu --> [Output]
+
+    Replaced with:
+    [Input] --> Mul --> Sigmoid --> Mul --> [Output]
+                 |                    ^
+                 |                    |
+                 +--------------------+
+
+    Where the first Mul multiplies by alpha (default 1.702).
+    QuickGelu(x) = x * sigmoid(alpha * x)
+    """
+
+    ALPHA = 1.702  # QuickGelu default alpha
+
+    def __init__(self):
+        super().__init__()
+        self._rule = pattern.RewriteRule(
+            self._target_pattern,
+            self._replacement_pattern,
+        )
+
+    def _target_pattern(self, op, x):
+        return op.QuickGelu(x, _domain=MSFT_DOMAIN)
+
+    def _replacement_pattern(self, op, x):
+        # Create alpha constant
+        x_dtype = x.dtype if x.dtype is not None else ir.DataType.FLOAT
+        alpha = op.Constant(value=ir.tensor(self.ALPHA, dtype=x_dtype))
```

Comment:
What if the graph already has this node?

Reply:
If op.Constant guarantee creating a unique name, this case won't happen.

Reply:
Done!